src/domain/proposal.ts

```diff
@@ -437,7 +437,7 @@ export function updateProposalExecution(
 ): void {
   let proposal = getProposal(proposalId.toHex());
   proposal.executedAt = timestamp;
-  proposal.closingAt = timestamp;
+  proposal.closingAt = timestamp.plus(BigInt.fromI32(1500000000));
```

Comment:
can we make is a const somehow ? 
add a comment : `Setting the closingAt field to a far away point in the future so it will be easy to sort all proposal(open and executed) in ascending order by the closingAt field`